plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/internal/impl/EditManagerImpl.java

```diff
@@ -695,11 +695,18 @@ public void refreshEditFeature() {
             Set<Identifier> fids = FeatureUtils.stringToId(filterFactory, getEditFeature().getID());
             Id filter = filterFactory.id(fids);
             FeatureIterator<SimpleFeature> features = resource.getFeatures(filter).features();
-            if (features.hasNext()) {
-                SimpleFeature feature = features.next();
-                setEditFeature(feature, editLayer);
-            } else {
-                setEditFeature(null, editLayer);
+            try 
```

Comment:
Looks like formating of try/catch is different to other code-parts. Are you using formatter https://github.com/locationtech/udig-platform/blob/master/extras/org.locationtech.udig.dev/codeformatter.xml in Eclipse? Please just format changed code elements by selecting lines and apply Right Mouse -> Source -> Formate Element

Reply:
Done, I'll be sure to use that from now on. :)